<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteViewer</class>
 <widget class="QDialog" name="PaletteViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>108</width>
      <height>108</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="redLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Red:</string>
   </property>
  </widget>
  <widget class="QLabel" name="greenLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Green:</string>
   </property>
  </widget>
  <widget class="QLabel" name="blueLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue:</string>
   </property>
  </widget>
  <widget class="QLabel" name="alphaLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Alpha</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="colorNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="redValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="blueValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="alphaValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="greenValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
